.github/workflows/canary-integration-test.yml

```diff
@@ -286,6 +285,13 @@ jobs:
       - name: wait for ceph to be ready
         run: tests/scripts/github-action-helper.sh wait_for_ceph_to_be_ready osd 2
 
+      - name: test toolbox-operator-image pod
+        run: |
+          tests/scripts/github-action-helper.sh create_operator_toolbox
+          toolbox=$(kubectl get pod -l app=rook-ceph-tools-operator-image -n rook-ceph -o jsonpath='{.items[*].metadata.name}')
+          timeout 120 sh -c "while kubectl get pod $toolbox -nrook-ceph -o jsonpath='{.items[*].status.phase}'|grep -qE 'Running'; do echo 'waiting for operator toolbox pod to start'; sleep 5; done"
```

Comment:
Checking is `|grep -qE 'Running'` works for validation.

But the pod actually runs fine